dockers/docker-fpm-frr/base_image_files/monit_bgp

```diff
@@ -21,3 +22,6 @@ check process staticd matching "/usr/lib/frr/staticd"
 
 check process bgpcfgd matching "python /usr/local/bin/bgpcfgd"
     if does not exist for 5 times within 5 cycles then alert
+
+check process bgpmon matching "python2 /usr/bin/bgpmon.py"
```

Comment:
>/usr/bin/bgpmon.py [](start = 39, length = 18)

Does the process name include "python2"? If yes, possible to remove?

Reply:
@qiluo-msft Per comment in https://github.com/Azure/sonic-swss/pull/1429 to make the daemon specific to python2, once that change is in place we see it gets reflected under the output of show services somethis as following:
```
admin@SONic:~$ show services
...
bgp     docker
---------------------------
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  5.2  0.2  28344 22532 pts/0    Ss+  03:47   0:03 /usr/bin/python2 /usr/bin/supervisord
root        21  0.7  0.2  22660 16684 pts/0    S    03:47   0:00 python /usr/bin/supervisor-proc-exit-listener --container-name bgp
root        25  0.1  0.0 225856  3488 pts/0    Sl   03:47   0:00 /usr/sbin/rsyslogd -n -iNONE
frr         29  0.8  0.1 503176 16156 pts/0    Sl   03:47   0:00 /usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M fpm -M snmp
frr         30  0.2  0.0  43176  6140 pts/0    S    03:47   0:00 /usr/lib/frr/staticd -A 127.0.0.1
frr         31  1.3  0.2 294536 20960 pts/0    Sl   03:47   0:00 /usr/lib/frr/bgpd -A 127.0.0.1 -M snmp
root        38  6.1  0.6  69172 55992 pts/0    S    03:47   0:03 /usr/bin/python /usr/local/bin/bgpcfgd
root        41  0.6  0.1  20996 15028 pts/0    S    03:47   0:00 python2 /usr/bin/bgpmon.py
...
```
For this reason the Monit config specification needs to match it.